src/doc/man/generated_txt/cargo-add.txt

```diff
@@ -171,16 +171,19 @@ OPTIONS
        -p spec, --package spec
            Add dependencies to only the specified package.
 
-       --frozen, --locked
-           Either of these flags requires that the Cargo.lock file be
-           up-to-date. If the lock file is missing, or it needs to be updated,
-           Cargo will exit with an error. The --frozen flag also prevents Cargo
-           from attempting to access the network to determine if it is
-           out-of-date.
+       --locked
+           Ensures that Cargo uses the exact version of every dependency
```

Comment:
Still not too happy with "Cargo **uses** the exact version of every dependency", which optional dependencies are not used. Suggestions?

Reply:
Not sure but "locking" sounds like Cargo will lock something but this is Cargo being locked to Cargo.lock